ProcessLib/LIE/Common/LevelSetFunction.cpp

```diff
@@ -10,14 +10,22 @@
 
 #include <boost/math/special_functions/sign.hpp>
 
+#include "BranchProperty.h"
 #include "FractureProperty.h"
+#include "JunctionProperty.h"
 
 namespace
 {
 // Heaviside step function
 inline double Heaviside(double v)
 {
-    return (v < 0.0) ? 0.0 : 1.0;
+	return (v < 0.0) ? 0.0 : 1.0;
+}
+
+template <class T_VEC, class T_V>
+inline bool inList(T_VEC const& vec, T_V const& v)
```

Comment:
Would suggest to move it to BaseLib/Algorithm.h, and rename to `contains` or similar, because it is not acting on lists only.